sat_vapor_pres/sat_vapor_pres.F90

```diff
@@ -535,7 +535,7 @@ module sat_vapor_pres_mod
 
 ! The default values below preserve the behavior of omsk and earlier revisions.
  logical :: show_bad_value_count_by_slice=.true.
- logical :: show_all_bad_values=.false.
+ logical :: show_all_bad_values=.true. ! Changed LMH 22may2018
```

Comment:
Is there universal acceptance for changing the default behavior of a namelist controlled variable?